### PR TITLE
Jose V.'s Fragment

### PR DESCRIPTION
**Pull request by JDVila**
Jose V.'s Fragment - It's okay, I guess....

### CODE REVIEW

File: app/src/main/java/nyc/c4q/grocerystore/employeeHandbook/EmployeeListAdapter.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 class EmployeeListAdapter extends RecyclerView.Adapter{
 
     private List<EmployeeDescription> mData = Arrays.asList(
-        new EmployeeDescription("Ashique", R.drawable.ashique)
+        new EmployeeDescription("JoseV", R.drawable.unnamed)
```
Comment (jsjrobotics) [inline app/src/main/java/nyc/c4q/grocerystore/employeeHandbook/EmployeeListAdapter.java:17]: @JDVila this drawable should have a descriptive name, not unamed

Comment (JDVila) [reply app/src/main/java/nyc/c4q/grocerystore/employeeHandbook/EmployeeListAdapter.java:17]: Okay, thank you for pointing that out! Refactored the image name to "josev"

**Comment by jsjrobotics** [comment]
merged in with other pr

**Comment by JDVila** [comment]
Excellent! Thank you!

**Pull request closed without merging**